pipeline/terraform/main.tf

```diff
@@ -92,7 +92,7 @@ module "catalogue_pipeline_20200727" {
   ]
 
   miro_adapter_topic_arns = [
-    //    local.miro_reindexer_topic_arn,
+    local.miro_reindexer_topic_arn,
```

Comment:
Presumably this is back on because with this change you want to be able to reindex miro things?

Reply:
It is mistake